site/docs/status.md

```diff
@@ -0,0 +1,358 @@
+| renameView     | Y    | N         | N    | N  |
+
+#### Namespace Operations
+
+| Namespace Operation       | Java | PyIceberg | Rust | Go |
+|---------------------------|------|-----------|------|----|
+| listNamespaces            | Y    | Y         | Y    | Y  |
+| createNamespace           | Y    | Y         | Y    | Y  |
+| dropNamespace             | Y    | Y         | Y    | Y  |
+| namespaceExists           | Y    | N         | Y    | Y  |
+| updateNamespaceProperties | Y    | Y         | Y    | Y  |
+| loadNamespaceMetadata     | Y    | N         | Y    | Y  |
+
+### Sql Catalog
+
+The sql catalog is a catalog backed by a sql database, which is called jdbc catalog in java.
+
+| Database | Java | PyIceberg | Rust | Go |
+|----------|------|-----------|------|----|
+| Postgres | Y    | Y         | Y    | N  |
+| MySQL    | Y    | Y         | Y    | N  |
+| SQLite   | Y    | Y         | Y    | N  |
+
+#### Table Spec V1
+
+| Table Operation | Java | PyIceberg | Rust | Go |
+|-----------------|------|-----------|------|----|
+| listTable       | Y    | Y         | Y    | Y  |
+| createTable     | Y    | Y         | Y    | Y  |
+| dropTable       | Y    | Y         | Y    | Y  |
+| loadTable       | Y    | Y         | Y    | Y  |
+| updateTable     | Y    | Y         | Y    | Y  |
+| renameTable     | Y    | Y         | Y    | Y  |
+| tableExists     | Y    | Y         | Y    | Y  |
+
+#### Table Spec V2
+
+| Table Operation | Java | PyIceberg | Rust | Go |
+|-----------------|------|-----------|------|----|
+| listTable       | Y    | Y         | Y    | Y  |
+| createTable     | Y    | Y         | Y    | Y  |
+| dropTable       | Y    | Y         | Y    | Y  |
+| loadTable       | Y    | Y         | Y    | Y  |
+| updateTable     | Y    | Y         | Y    | Y  |
+| renameTable     | Y    | Y         | Y    | Y  |
+| tableExists     | Y    | Y         | Y    | Y  |
+
+#### View Spec V1
+
+| View Operation | Java | PyIceberg | Rust | Go |
+|----------------|------|-----------|------|----|
+| createView     | Y    | N         | N    | N  |
+| dropView       | Y    | N         | N    | N  |
+| listView       | Y    | N         | N    | N  |
+| viewExists     | Y    | N         | N    | N  |
+| replaceView    | Y    | N         | N    | N  |
+| renameView     | Y    | N         | N    | N  |
+
+#### Namespace Operations
+
+| Namespace Operation       | Java | PyIceberg | Rust | Go |
+|---------------------------|------|-----------|------|----|
+| listNamespaces            | Y    | N         | N    | N  |
+| createNamespace           | Y    | N         | N    | N  |
+| dropNamespace             | Y    | Y         | Y    | Y  |
+| namespaceExists           | Y    | N         | N    | N  |
+| renameNamespace           | Y    | N         | N    | N  |
+| updateNamespace           | Y    | N         | N    | N  |
+| updateNamespaceProperties | Y    | Y         | Y    | Y  |
+| loadNamespaceMetadata     | Y    | N         | N    | N  |
```

Comment:
typo/duplicate as I don't think there's an `updateNamespace` that's different from `updateNamespaceProperties`, and `renameNamespace` isn't a function in the JdbcCatalog:
```suggestion
| updateNamespaceProperties | Y    | Y         | Y    | Y  |
| loadNamespaceMetadata     | Y    | N         | N    | N  |
```
https://github.com/apache/iceberg/blob/540d6a6251e31b232fe6ed2413680621454d107a/core/src/main/java/org/apache/iceberg/jdbc/JdbcCatalog.java#L809